py3status/modules/velib_metropole.py

```diff
@@ -0,0 +1,292 @@
+# -*- coding: utf-8 -*-
+"""
+Display Velib shared bike avaibility on our favorite stations.
+Use https://www.velib-metropole.fr/map data.
+
+You only need to set id of stations you want to monitor in station_codes param.
+Then scroll to cycle stations, and middle clic to force a refresh.
+
+Configuration parameters:
+    button_next: Display next station (default 5)
+    button_previous: Display previous station (default 4)
+    button_refresh: refresh stations (default 2)
+    cache_timeout: The time between API polling in seconds
+        It is recommended to keep this at a higher value to avoid rate
+        limiting with the API's. (default 60)
+    format: How to display the velib data.
+        (default 'Velib Metropole: {index}/{stations} {format_station}')
+    format_station: How to display the velib station data.
+        *(default '{station_name}: [\?color=station_state_code {station_state}]'
+            '[\?soft  ][\?color=greenyellow {nb_bike}/{nb_free_e_dock}]'
+            '[\?soft  ][\?color=deepskyblue {nb_ebike}/{nb_free_e_dock}]')*
+    station_codes: List of velib stations to monitor.
+        You can get stations id on map here https://www.velib-metropole.fr/map
+        (default [20043, 11014, 20012, 20014, 10042])
+    thresholds: Configure colors of format station.
+        (default {'station_state_code': [(0, 'good'), (1, 'bad')]})
```

Comment:
Default with `[(0, 'good'), (1, 'bad')]` instead. If this module does not work with that option, then your threshold code is probably a little off.